miio/airconditioningcompanion.py

```diff
@@ -166,50 +169,104 @@ def __repr__(self) -> str:
              self.mode)
         return s
 
+    def __json__(self):
+        return self.data
+
 
 class AirConditioningCompanion(Device):
     """Main class representing Xiaomi Air Conditioning Companion."""
 
+    @command(
+        default_output=format_output(
+            "",
+            "Power: {result.power}\n"
+            "Load power: {result.load_power}\n"
+            "Air Condition model: {result.air_condition_model}\n"
+            "LED: {result.led}\n"
+            "Target temperature: {result.target_temperature} °C\n"
+            "Swing mode: {result.swing_mode}\n"
+            "Fan speed: {result.fan_speed}\n"
+            "Mode: {result.mode}\n"
+        )
+    )
     def status(self) -> AirConditioningCompanionStatus:
         """Return device status."""
         status = self.send("get_model_and_state", [])
         return AirConditioningCompanionStatus(status)
 
+    @command(
+        default_output=format_output("Powering the air condition on"),
+    )
```

Comment:
When it all fits into one line (and there are no arguments), I think it'd be nice to have it all on a single line.

Reply:
Are you sure? It feels hard to read because of the braces and the little spaces.

Reply:
Well, I think it would be nicer for grepping the code, but it's not so important and can be changed later if needed.